ui/components/multi-select-wrapper.js

```diff
@@ -179,41 +186,46 @@ const MultiSelectWrapper = (props) => {
       ...base,
       backgroundColor: base.backgroundColor2,
     }),
+    input: (base) => ({
+      ...base,
+      color: theme.palette.text.primary,
```

Comment:
Also make sure this input is a sistent component

Reply:
I attempted to use the Input component from @layer5/sistent instead of the default Input from react-select. However, this approach did not work well because the Sistent's Input component cannot handle all the props passed by react-select which are needed for the Input component to function correctly. While the theme works when i do this there is a loss of functionality. Let me know what you think of this.